### PR TITLE
i18n: Update es-AR translations

### PR DESCRIPTION
**Pull request by tombettany**
Update the `es-AR` translations.

cc @skarbat

### CODE REVIEW

File: po/es_AR.po [hunk 1]
```diff
@@ -951,138 +682,488 @@ msgstr "Escoge tu fondo"
...
+#~ msgid "Unpairing {}"
+#~ msgstr "Desemparejando {}"
+
+#~ msgid "Pairing {}"
+#~ msgstr "Emparejando {}"
 
-msgid "Get ready... Kano is starting up.\n"
-msgstr "Prepárate... Kano está empezando.\n"
+#~ msgid "Get ready... Kano is starting up.\n"
+#~ msgstr "Prepárate... Kano está empezando.\n"
```
Comment (tombettany) [inline po/es_AR.po]: @skarbat Did we manually add this one last time?

**Comment by tombettany** [comment]
[Settings](https://trello.com/c/gKG9ZhOp/16-settings)

**Comment by skarbat** [comment]
This is correct @tombettany - I did that because we needed a way to pull the translation from a script:

 * https://github.com/KanoComputing/kano-settings/blob/master/etc/init.d/kano-settings#L17

**Comment by Ealdwulf** [comment]
That can be automated like this:
https://github.com/KanoComputing/make-snake/blob/master/po/Makefile#L27, I
think.

**Comment by skarbat** [comment]
@tombettany, Just pushed 3 small translation typos - Looks good to me.

**Comment by tombettany** [comment]
@Ealdwulf I made your suggested changes. Merging now for testing.

File: etc/init.d/kano-settings [hunk 2]
```diff
@@ -10,11 +10,13 @@
 ### END INIT INFO
 
 . /lib/lsb/init-functions
+. gettext.sh
```
Comment (skarbat) [inline etc/init.d/kano-settings]: should this have an absolute path, given it's pulled from a former sysv init script?

Comment (tombettany) [inline etc/init.d/kano-settings]: Hmm, we'll try it and see what happens...

**Pull request merged**